src/Octoshift/Model/Mannequin.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+namespace OctoshiftCLI.Data
```

Comment:
Since we changed the directory name to `Model`,the namespace needs to also change:
```suggestion
namespace OctoshiftCLI.Model
```

Also I think it'd be better to use a plural noun (`Models`) for the folder name 😃

Reply:
Fixed [1f3dd](https://github.com/github/gh-gei/pull/271/commits/1f3ddaf6696e8f19aa62b7117105ac6b78ad6560)